packages/slate-react/src/components/leaf.tsx

```diff
@@ -50,20 +31,17 @@ const Leaf = (props: {
     children = (
       <React.Fragment>
         <span
-          ref={placeholderRef}
           contentEditable={false}
           style={{
             pointerEvents: 'none',
             display: 'inline-block',
             width: '100%',
             maxWidth: '100%',
-            whiteSpace: 'nowrap',
             opacity: '0.333',
             userSelect: 'none',
             fontStyle: 'normal',
             fontWeight: 'normal',
             textDecoration: 'none',
-            position: 'absolute',
```

Comment:
removing this breaks the placeholder because it will now be rendered below the first empty line rather than in the same place

Reply:
Hi @juliankrispel, could you explain why?
This is demo with current react-slate: https://codesandbox.io/s/clever-pare-ywluq?file=/src/App.js
This is demo with this PR: https://codesandbox.io/s/pensive-noyce-q83gr?file=/src/App.js